android-sdk/src/main/java/com/mobilecoin/lib/MobileCoinClient.java

```diff
@@ -971,9 +976,7 @@ public synchronized void shutdown() {
         if (null != blockchainClient) {
             blockchainClient.shutdown();
         }
-        if (null != untrustedClient) {
-            untrustedClient.shutdown();
-        }
+        getUntrustedClient().shutdown();
```

Comment:
`untrustedClient` can be null according to the package-private constructor, so this could crash